Makefile

```diff
@@ -317,7 +317,9 @@ $(HELMIFY): $(LOCALBIN)
 
 .PHONY: helm
 helm: manifests kustomize helmify
-	$(KUSTOMIZE) build config/default | $(HELMIFY) -crd-dir
+	$(KUSTOMIZE) build config/default \
+	| yq 'select(.kind and (.kind != "ConfigMap" or .metadata.name != "llmaz-global-config"))' \
```

Comment:
Better not to add too much logic to the makefile command, what's the advantages over the current one?

Reply:
Because the default value is `default-scheduler` in `config/default/configmap.yaml` and it is can not be easily changed by user via `helm install --set` command. 

I can not edit `chart/template/global-config.yaml` because it is generated by helmify. I want to user can easily use chart without extra config.

Reply:
But it's not a general solution, we have a lot of configurations.

Reply:
Can we switch the default scheduler to `inftyai-scheduler` in `config/default/configmap.yaml`? And add `make install-inftyai-scheduler` and `make uninstall-inftyai-scheduler` to the Makefile.

```
install-inftyai-scheduler:
	kubectl apply --server-side -k config/inftyai-scheduler

.PHONY: uninstall-prometheus
uninstall-inftyai-scheduler:
	kubectl delete -k config/inftyai-scheduler
```

Reply:
If not, we can not enable the `inftyai-scheduler` in chart by default due to the immutability of the configmap.

Reply:
> I'm ok to let user configure the configmaps via global.values.yaml. It's also not difficult.

This pr has supported it.